src/header.wppl

```diff
@@ -531,7 +531,7 @@ var Vector = function(arr) {
 var Matrix = function(arr) {
   var n = arr.length;
   var m = arr[0].length;
-  var t = ad.tensor.fromScalars(_.flatten(arr));
+  var t = ad.tensor.fromScalars(_.flattenDeep(arr));
```

Comment:
Do we only need `_.flatten` here? I think the assumption is that the input array is a list of lists, e.g. `[[1, 2], [3, 4]]`.